receiver/prometheusreceiver/internal/metricfamily.go

```diff
@@ -144,6 +147,7 @@ func (mg *metricGroup) toDistributionPoint(dest pmetric.HistogramDataPointSlice)
 	point.SetTimestamp(tsNanos)
 	populateAttributes(pmetric.MetricTypeHistogram, mg.ls, point.Attributes())
 	mg.setExemplars(point.Exemplars())
+	point.CopyTo(dest.AppendEmpty())
```

Comment:
IIRC, CopyTo is expensive.  Can we avoid adding this?

Reply:
We can use `MoveTo` instead, sure.

Reply:
Done.